assets/src/editor/events/dates-and-times/editor-date/grid-view/event-date-details-panel.js

```diff
@@ -7,68 +7,56 @@ import { parseInfinity } from '@eventespresso/utils';
 import { __ } from '@eventespresso/i18n';
 import { InfinitySymbol } from '@eventespresso/value-objects';
 import PropTypes from 'prop-types';
-import { useEventDateUpdateRelatedTickets } from '@eventespresso/hooks';
 
 /**
  * Internal dependencies
  */
 import EventDateRegistrationsLink from '../event-date-registrations-link';
-import { updateTicketQtyAfterCapacityChange } from '../../../mutations/ticket-mutations';
 
-const EventDateDetailsPanel = ( { eventDate } ) => {
-	const updateRelatedTickets = useEventDateUpdateRelatedTickets( eventDate );
-	return useMemo(
-		() => {
-			const details = [
-				{
-					id: 'ee-event-date-sold',
-					label: __( 'sold', 'event_espresso' ),
-					value: eventDate.sold || 0,
-				},
-				{
-					id: 'ee-event-date-reserved',
-					label: __( 'reserved', 'event_espresso' ),
-					value: eventDate.reserved || 0,
-				},
-				{
-					id: 'ee-event-date-capacity',
-					label: __( 'capacity', 'event_espresso' ),
-					value: <InfinitySymbol value={ eventDate.regLimit } asInt />,
-					editable: {
-						type: 'text',
-						valueType: 'infinite',
-						onChange: ( cap ) => {
-							const capacity = parseInfinity( cap, true, true );
-							eventDate.regLimit = capacity;
-							const updateTicketQty = updateTicketQtyAfterCapacityChange( capacity );
```

Comment:
uh-oh... did this get lost?

Reply:
🤦‍♂

Reply:
Pushed the fix. Thank you for being an eagle eyed reviewer 👍